robot_ws/src/hello_world_robot/package.xml

```diff
@@ -1,27 +1,21 @@
 <?xml version="1.0"?>
-<package format="2">
+<package format="3">
   <name>hello_world_robot</name>
-  <version>1.2.0</version>
+  <version>1.0.0</version>
   <description>
     AWS RoboMaker robot package with a rotating Turtlebot3
   </description>
   <license>MIT</license>
   <author email="ros-contributions@amazon.com">AWS RoboMaker</author>
   <maintainer email="ros-contributions@amazon.com">AWS RoboMaker</maintainer>
-  <buildtool_depend>catkin</buildtool_depend>
-  <depend>rospy</depend>
-  <depend>actionlib</depend>
-  <depend>interactive_markers</depend>
-  <depend>std_msgs</depend>
-  <depend>sensor_msgs</depend>
+  <buildtool_depend>ament</buildtool_depend>
+  <depend>rclpy</depend>
   <depend>geometry_msgs</depend>
-  <depend>nav_msgs</depend>
-  <depend>visualization_msgs</depend>
-  <depend>actionlib_msgs</depend>
-  <depend>turtlebot3</depend>
```

Comment:
Curious how come we don't need the turtlebot3 dependencies here? were they just redundant in the ROS1 version?

Reply:
Yes, I believe these were never needed for this package.